packages/grafana-ui/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FunctionComponent } from 'react';
+import { Range, createSliderWithTooltip } from 'rc-slider';
+import { css } from 'emotion';
+import { stylesFactory } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes/ThemeContext';
+
+export interface Props {
+  min: number;
+  max: number;
+  orientation: 'horizontal' | 'vertical';
+  // Set current positions of handle(s). If only 1 value supplied, only 1 handle displayed.
+  value?: number[];
+  reverse?: boolean;
+  lengthOfSlider?: number;
```

Comment:
I'm not a huge fan of statically defined length. Can we instead make the slider be always 100% and adjust the width with the container element when necessary?

Reply:
Yeah, you are right. I was thinking about it, but then I've decided to add defined length. I have changed it to 100%.

Reply:
thank you 🙏